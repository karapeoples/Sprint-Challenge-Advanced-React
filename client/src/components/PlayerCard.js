import React from 'react';


const PlayerCard = ({name, location, searchInfo}) => {
  

  return (
    <div>
      <h1>Name: {name} </h1>
      <h3>Country: {location}</h3>
      <p>Peoples have searched for this player {searchInfo} times.</p>

    </div>
  )
}

export default PlayerCard;