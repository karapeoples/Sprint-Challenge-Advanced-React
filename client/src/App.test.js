import React from 'react';
import * as rtl from "@testing-library/react";
import PlayerCard from './components/PlayerCard';
import renderer from 'react-test-renderer';


test('Testing Test', () => {
  expect(true).toBeTruthy()
})


//Test 1
test('Test for Labels', () => {
  const { getByText } = rtl.render(<PlayerCard />);
  getByText(/name/i);
  getByText(/country/i);
});


//Test 2
test('renders correctly when there are no items', () => {
  const tree = renderer.create(<PlayerCard />).toJSON();
  expect(tree).toMatchSnapshot();
});

//Test 3
//Could not implement a third test due to lack of information in the sprint and lack of knowledge to test API and Toggle Methods. To many variables and things to add to make either of those tests work but I do have knowledge of them and think I can implement those types of tests as I learn more.

