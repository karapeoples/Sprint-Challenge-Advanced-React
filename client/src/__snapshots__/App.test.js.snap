// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly when there are no items 1`] = `
<div>
  <h1>
    Name: 
     
  </h1>
  <h3>
    Country: 
  </h3>
  <p>
    Peoples have searched for this player 
     times.
  </p>
</div>
`;
